web/src/components/tasks/business-formation/contacts/Addresses.tsx

```diff
@@ -186,7 +187,18 @@ export const Addresses = <T extends FormationMember | FormationIncorporator>(
       <tfoot>
         {props.hasError && props.addressData.length === 0 ? (
           <tr>
-            <td className={"text-error-dark text-bold"}>{props.displayContent.error}</td>
+            <td colSpan={3} className={"text-error-dark text-bold"}>
+              {props.displayContent.error}
+            </td>
+          </tr>
+        ) : props.hasError &&
+          props.fieldName === "members" &&
+          props.legalType === "nonprofit" &&
+          props.addressData.length < 3 ? (
```

Comment:
[boulder] This logic should not be necessary (here and further down in this file). This should be able to be controlled solely by `props.hasError`.

At a code design level, we know you already coded the logic for the fact that "members" has an error for fewer than 3 for nonprofits, and for fewer than 1 for not nonprofits. You tested that explicitly in `getErrorStateForField.test.ts` because the logic lives right there in `getErrorStateForField`. So at a code design level, when you are rewriting error handling logic that has already been logically defined elsewhere, that should be an alarm bell.

From a code perspective, we can trace this back. `props.hasError` comes here into `<Addresses>` from `<Members>`. Taking a look at `Members`, I can see on line 69 that the prop value comes from *its* props. Okay, who renders Members? Looks like now-line-49 of `ContactsStep`. And the value of that prop therefore comes from `doesFieldHaveError("members")`. And if we trace backward into how `doesFieldHaveError` makes its decision (inside of `useFormationErrors`) we can see that value is calculated in `getErrorStateForField` - which of course, already handles the logical decision about the minimum length being 3 or 1 depending on legalType!

So far as I can see, redoing this logic here should not be necessary (and thus, neither is adding the new legalType prop)